conf/celeryconfig.py

```diff
@@ -5,17 +5,27 @@
 from celery.schedules import crontab
 
 from chronam.settings import *  # NOQA
-from chronam.settings import INSTALLED_APPS
 
 APP_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 
 
-CELERY_IMPORTS = ("chronam.core.tasks",)
-CELERYD_LOG_FILE = os.path.join("/logs/celery", "celery.log")
+CELERY_IMPORTS = ("chronam.core.tasks", "chronam.loc_cts.tasks")
```

Comment:
This is an internal-only task. We probably need to make this configurable so we can enable it for only on the servers where that's relevant.